lib/config_default.js

```diff
@@ -200,9 +200,21 @@ export const config = {
     "AP121U",
     "D-Link DIR-615",
     "D-Link DIR-615 D",
+    "D-Link DIR-825",
```

Comment:
Why? This has been fixed and the limit only applies to factory installations (Which are removed from Gluon)

Reply:
I went through the list of targets of the latest release and it was marked as only 6M flash 
https://github.com/freifunk-gluon/gluon/blob/7feebe96a6b93d567bd342ffd01abcb3022383c5/targets/ath79-generic#L124

Reply:
This is outdated then, see https://github.com/openwrt/openwrt/commit/aca8bb5cc332f0ffdf4249e76b0a56716f98bef0